### PR TITLE
🌐 Add German translation for `docs/de/docs/how-to/extending-openapi.md`

### PR DESCRIPTION
**Pull request by nilslindemann**
← `how-to/conditional-openapi.md` (#10790)
→ `how-to/separate-openapi-schemas.md` (#10796)

[German translation progress](https://github.com/tiangolo/fastapi/discussions/10582)

### CODE REVIEW

**Comment by tiangolo** [comment]
📝 Docs preview for commit c2f266289502edc3c6d03817f400afe938e0ad5f at: https://2318bdcb.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit 5736fd9bee3822908ad737bc2791f82a63516f26 at: https://7bef6141.fastapitiangolo.pages.dev

**Comment by tiangolo** [comment]
📝 Docs preview for commit df61b01981405e8ea495faa7289717fed54aff1c at: https://bb3998a8.fastapitiangolo.pages.dev

**Review by dpinezich — CHANGES_REQUESTED** [review]
Awesome work, small changes (ideas) as always 👍

File: docs/de/docs/how-to/extending-openapi.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+
+Eine `FastAPI`-Anwendung (-Instanz) verfügt über eine `.openapi()`-Methode, von der erwartet wird, dass sie das OpenAPI-Schema zurückgibt.
+
+Als Teil der Erstellung des Anwendungsobjekts wird eine *Pfadoperation* für `/openapi.json` (oder was auch immer der Wert von `openapi_url` ist) registriert.
+
+Diese gibt lediglich eine JSON-Response zurück, welche das Ergebnis der Methode `.openapi()` der Anwendung ist.
+
+Standardmäßig überprüft die Methode `.openapi()` die Eigenschaft `.openapi_schema`, um zu sehen, ob diese Inhalt enthält, und gibt diesen zurück.
+
+Ist das nicht der Fall, wird er mithilfe der Hilfsfunktion unter `fastapi.openapi.utils.get_openapi` generiert.
```
[the remark this answers is not in the document]
Comment (nilslindemann) [reply docs/de/docs/how-to/extending-openapi.md:19]: Ja, ist klarer.

**Comment by nilslindemann** [comment]
Danke wieder, @dpinezich!

**Review by dpinezich — APPROVED** [review]
Well done 👍

**Comment by alejsdev** [comment]
Thanks for the help! @nilslindemann 🚀 @dpinezich 🔍️

**Comment by nilslindemann** [comment]
Super, wieder eins mehr :-)

**Pull request merged**